docs/themerc

```diff
@@ -71,3 +71,12 @@ osd.window-switcher.item.active.border.width: 2
 
 osd.workspace-switcher.boxes.width: 20
 osd.workspace-switcher.boxes.height: 20
+
+snapping-preview.region.fill: auto
+snapping-preview.edge.fill: auto
+snapping-preview.region.bg.color: #8080b380
+snapping-preview.edge.bg.color: #8080b380
+snapping-preview.region.border.width: 1
+snapping-preview.edge.border.width: 1
+snapping-preview.region.border.color: #ffffff,#000000,#ffffff
+snapping-preview.edge.border.color: #ffffff,#000000,#ffffff
```

Comment:
Do we want to change the default to white-black-white here?
Using the OSD colors instead as default will somewhat match the theme in use and should also provide a high enough contrast ratio (as otherwise the text on the OSD wouldn't be readable).

Reply:
That's right.
I didn't know that some OSD styles already falls back to values like `window.active.border.color` when I made this change, and thought that I need to allow any dedicated value like `auto` to make it fall back, which sounds tedious.

I will fix it soon.